### PR TITLE
PR for Notes form, rich text editor setup

### PR DESCRIPTION
**Pull request by manismk**
- Added Notes Form
- Added Rich text editor
- Added basic notes context
- [Link](https://deploy-preview-5--key-notes.netlify.app/app/notes)

`Note: Still not storing the data in db`

### CODE REVIEW

**Review by AnjaliDeshwani — APPROVED** [review]
Clean code and good folder structure, Changes looks good to merge.

File: src/components/notes-form/notesForm.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.form--container {
+  z-index: 30;
```
Comment (znjs) [inline src/components/notes-form/notesForm.css:2]: `z-index` won't work on `display:static` which is the default.
Kindly add a `position` property

File: src/reducer/notesReducer.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export const notesReducer = (state, action) => {
+  switch (action.type) {
+    case "ADD_NOTES":
```
Comment (znjs) [inline src/reducer/notesReducer.js]: Suggestion: can use variables instead of strings to avoid typos.

**Review by znjs — CHANGES_REQUESTED** [review]
Good folder structures
Liked that you explored mui and ReactQuill

File: src/components/notes-form/notesForm.css (hunk 1 above)
Comment (manismk) [reply src/components/notes-form/notesForm.css:2]: Thanks @znjs  for the review. Here I have used the position property in the component library's modal class. Here only I am increasing the Z- index value.

File: src/reducer/notesReducer.js (hunk 2 above)
Comment (manismk) [reply src/reducer/notesReducer.js]: Thank for pointing it out, Made the changes

**Pull request merged**